packages/app/src/components/Button/Button.tsx

```diff
@@ -167,6 +168,13 @@ const BaseButton = styled.button<BaseButtonProps>`
 				display: none;
 			}
 		`}
+	
+	${(props) =>
+		props.$variant === 'yellow' &&
+		props.outlineColor === 'yellow' &&
```

Comment:
Would really like to avoid adding new props to this component, I know it's a bit messy currently anyway but the idea is that the parent should only need to pass a variant and then we have the different variant styles defined inside here. Would probably be better if we just created a variant for this particular style `staking-button` or something?

Reply:
Thanks for the advice. Fixed.